llvm/cmake/config-ix.cmake

```diff
@@ -209,18 +209,6 @@ if(LLVM_ENABLE_ZSTD)
   endif()
 endif()
 
-# If LLVM_USE_STATIC_ZSTD is specified, make sure we enable zstd only if static
-# libraries are found.
-if(LLVM_USE_STATIC_ZSTD AND NOT TARGET zstd::libzstd_static)
-  # Fail if LLVM_ENABLE_ZSTD is FORCE_ON.
-  if(LLVM_ENABLE_ZSTD STREQUAL FORCE_ON)
-      message(FATAL_ERROR "Failed to find static zstd libraries, but LLVM_USE_STATIC_ZSTD=ON and LLVM_ENABLE_ZSTD=FORCE_ON.")
-  endif()
-set(LLVM_ENABLE_ZSTD OFF)
-else()
-set(LLVM_ENABLE_ZSTD ${zstd_FOUND})
-endif()
```

Comment:
This code block is duplicate of the block above. Ideally, it should have been removed in the pulldown commit: https://github.com/intel/llvm/commit/8cb519c35654f869380d01ba2c19fa012a93b35c